pkg/didmethod/peer/generate.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package peer
+
+import (
+	"crypto"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+)
+
+const numalgo = "1"
```

Comment:
Better to have some comments on these as it's difficult to read the code (names doesn't give much info). Also, these 2 constants can be wrapped.

Reply:
Added comments and wrapped the constants

Reply:
Also, should be camel cased.

Reply:
done